docs/src/configuration/services/rabbitmq.md

```diff
@@ -66,36 +66,51 @@ highlight=python
 
 {{< /codetabs >}}
 
-(The specific way to inject configuration into your application will vary. Consult your application or framework's documentation.)
+(The specific way to inject configuration into your application varies.
+Consult your application or framework's documentation.)
 
 ## Connecting to RabbitMQ
 
 ### From your local development environment
 
-For debugging purposes, it's sometimes useful to be able to directly connect to a service instance. You can do this using SSH tunneling. To open a tunnel, log into your application container like usual, but with an extra flag to enable local port forwarding:
+For debugging purposes, it's sometimes useful to connect directly to a service with an SSH tunnel.
+To open a tunnel, follow these steps.
 
-```bash
-ssh -L 5672:rabbitmqqueue.internal:5672 <projectid>-<branch_ID>@ssh.eu.platform.sh
-```
+1. Get the username and password to the service by checking your app's relationships:
```

Comment:
Hm I haven't tried specifically for rabbitmq but I think tunnel:single should output relationship information like this on its own

Reply:
All I get is the following:

```bash
SSH tunnel opened to rabbitmq at: amqp://guest:guest@127.0.0.1:30000

Quitting this command (with Ctrl+C or equivalent) will close the tunnel.
```

Reply:
I missed that in the provided URI. :facepalm: